src/core/Akka/Actor/Scheduler/HashedWheelTimerScheduler.cs

```diff
@@ -542,13 +543,13 @@ public void Dispose()
         private sealed class ScheduledTell : IRunnable
         {
             private readonly ICanTell _receiver;
-            private readonly object _message;
+            private readonly IScheduledTellMsg _message;
             private readonly IActorRef _sender;
 
             public ScheduledTell(ICanTell receiver, object message, IActorRef sender)
             {
                 _receiver = receiver;
-                _message = message;
+                _message = message is INotInfluenceReceiveTimeout ? new ScheduledTellMsgNoInfluenceReceiveTimeout(message) : new ScheduledTellMsg(message);
```

Comment:
What if we do a check here to see if this `ActorRef` has no `ActorCell` and we unwrap the message automatically here?

Reply:
done